jenkins/scripts/files/run_fetch_logs.sh

```diff
@@ -14,6 +14,24 @@ else
 fi
 mkdir -p ${LOGS_DIR}
 
+# Fetch manifests
+declare -a resource=("bmh" \
+"machine" \
+"m3m" \
+"cluster" \
+"m3cluster" \
+"node")
+
+for object in "${resource[@]}"; do
+    mkdir -p "${LOGS_DIR}/${object}"
```

Comment:
can you make it under  another directory, like "objects"  or anything similar ? to avoid adding many folders in the root of the archive

Reply:
I created a folder `manifest`, and adding everything under it. Let me if you think it is not good idea.